apps/server/src/modules/transfer/import/imports/extract-actions.ts

```diff
@@ -0,0 +1,38 @@
+import { FlogoAppModel } from '@flogo-web/core';
+import {
+  ImportsActionsManager,
+  parseResourceIdFromResourceUri,
+} from '@flogo-web/lib-server/core';
+
+type ApplicationAction = FlogoAppModel.Action;
+
+export class ExtractActions implements ImportsActionsManager {
```

Comment:
nitpick: class name should be a noun => `ActionExtractor`

Reply:
Renamed the class to `FlogoActionExtractor`